fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1177,6 +1177,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, masterOnly = false)
     public static int cache_result_max_row_count = 3000;
 
+    /**
+     * Set the maximum data size that can be cached
+     */
+    @ConfField(mutable = true, masterOnly = false)
```

Comment:
Please add `description` field of `@ConfField`

Reply:
fixed